### PR TITLE
Update some packages and the README

### PR DESCRIPTION
**Pull request by arkon**
Updated a few of the dependencies, along with some minor edits to the README:

- Just use "ES2015" since that's the official name
- Capitalize "Babel" everywhere
- Fixed the part in the local dev section where it was referring to a `src` folder instead of just the root of the project

### CODE REVIEW

**Comment by souldreamer** [comment]
A lot of people still refer to it as ES6 and search for it as such, so it would probably be better for search optimization to leave it as ES6

**Comment by arkon** [comment]
@souldreamer I suppose so. Should I remove "ESNext" though? I haven't heard to it referred to as such in quite a while, especially with ES7/ES2016 around the corner.

**Comment by richgilbank** [comment]
Amazing, thanks @arkon! Yeah, I think a lot of people find it through the Chrome webstore by searching for "ES6", so maybe keep it in for the first mention of it in the Readme and add it in the Chrome manifest.

File: README.md [hunk 1]
```diff
@@ -1,34 +1,39 @@
 ## Scratch JS
 
 ### What is it?
-It's a Chrome DevTools extension ([available here](https://chrome.google.com/webstore/detail/scratch-js/alploljligeomonipppgaahpkenfnfkn)) that allows you to execute ES6/ESNext/ES2015 code in the context of the page you're viewing, as though it were the standard DevTools console.
+It's a Chrome DevTools extension ([available here](https://chrome.google.com/webstore/detail/scratch-js/alploljligeomonipppgaahpkenfnfkn)) that allows you to execute ES2015 code in the context of the page you're viewing, as though it were the standard DevTools console.
```
Comment (richgilbank) [inline README.md]: `ES6 / ES2015`

File: manifest.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Scratch JS",
   "version": "0.0.21",
-  "description": "Write and execute ES6 within DevTools!",
+  "description": "Write and execute ES2015 within DevTools!",
```
Comment (richgilbank) [inline manifest.json]: ES6 / ES2015

File: manifest.json [hunk 3]
```diff
@@ -14,7 +14,8 @@
     "tabs"
   ],
   "background": {
-    "scripts": ["background.js"]
+    "scripts": ["background.js"],
+    "persistent": false
```
Comment (richgilbank) [inline manifest.json]: :heart_eyes_cat:

**Comment by arkon** [comment]
@richgilbank Added back "ES6"!

**Comment by richgilbank** [comment]
Thanks again!

**Pull request merged**